### PR TITLE
Home Page added.

### PR DESCRIPTION
**Pull request by Helly-p**
Flipkart-Clone Home Page added.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,455 @@
...
+                                <a href="#"><i class="fa fa-shopping-cart" aria-hidden="true"
+                                        style="padding-right: 5px;font-size: 15px;"></i>Cart</a>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </header>
+    <br />
```
Comment (rutvik-simformsolutions) [inline index.html]: Use of "br" tag is not good practise.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,455 @@
...
+        <br>
+    </div>
+
+    <br />
+    <!-- Products -->
+    <div class="container-fluid" id="products">
+        <div class="row" style="padding-top: 10px;">
+            <div class="col-md-8">
+                <h5>Deals of the Day <span
+                        style="padding-left: 30px;font-size: 16px;font-weight: light;color: rgb(139, 139, 139);"><i
```
Comment (rutvik-simformsolutions) [inline index.html]: Remove Inline CSS

File: index.html [hunk 3]
```diff
@@ -0,0 +1,455 @@
...
+                    </ol>
+                    <div class="carousel-inner">
+                        <div class="item active">
+                            <div class="row">
+                                <div class="col-sm-2">
+                                    <div class="thumb-wrapper">
+                                        <div class="img-box"><img
+                                                src="https://rukminim1.flixcart.com/image/200/200/kokdci80/dslr-camera/v/e/x/z-24-200mm-z5-nikon-original-imag2zuekuxgxsgg.jpeg?q=70"
+                                                class="img-responsive" alt=""></div>
+                                        <div class="card-body" style="text-align: center;font-weight: lighter;">
```
Comment (rutvik-simformsolutions) [inline index.html]: remove inline css

File: index.html [hunk 4]
```diff
@@ -0,0 +1,455 @@
...
+                        <div class="card-body" style="text-align: center;font-weight: lighter;">
+                            <h5 class="card-title">Harmoniums</h5>
+                            <h5 class="price">Upto 60% Off</h5>
+                            <p>Shop Now!</p>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-sm-2">
+                    <div class="thumb-wrapper">
+                        <div class="img-box"><img
```
Comment (rutvik-simformsolutions) [inline index.html]: code is not proper formated

File: index.html [hunk 5]
```diff
@@ -0,0 +1,455 @@
...
+    <br />
+
+    <!-- Advertisement -->
+    <div class="container-fluid">
+        <div class="row">
+            <div id="carouselExampleControls" class="carousel slide" data-bs-ride="carousel">
+                <div class="carousel-inner">
+                    <div class="carousel-item active">
+                        <img src="https://rukminim1.flixcart.com/fk-p-flap/3376/560/image/a662a1f8e1374543.jpg?q=50"
+                            class="d-block w-100" alt="...">
```
Comment (rutvik-simformsolutions) [inline index.html]: put proper value in alt attribute rather than just using "..."

File: style.css [hunk 6]
```diff
@@ -0,0 +1,435 @@
...
+  padding: 0%;
+  box-sizing: border-box;
+  --bg-blue: #2874f0; 
+  font-family: 'Roboto', sans-serif;
+  letter-spacing: 1;
+}
+/* Header */
+header {
+  height: 56px;
+  background-color: var(--bg-blue);
```
Comment (rutvik-simformsolutions) [inline style.css:15]: Please explain this var() function in CSS

**Review by rutvik-simformsolutions — CHANGES_REQUESTED** [review]
Overall try to make the code more formatted.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,455 @@
...
+    <title>Online Shopping Site for Mobile, Electronics, CLothing Brands</title>
+
+    <!-- Favicon -->
+    <link rel="icon" href="./images/favicon.png" type="image/x-icon">
+
+    <!-- Font Awesome Icons -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+    <!-- Boostrap CSS & JS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
```
Comment (rutvik-simformsolutions) [inline index.html]: Instead of providing cdn here, we can practice placing it locally.

File: style.css (hunk 6 above)
Comment (Helly-p) [reply style.css:15]: It is used for accessing CSS variables. Instead of using same css property over and over again it is good to use CSS variables

**Review by rutvik-simformsolutions — APPROVED** [review]

**Pull request merged**